app.js

```diff
@@ -1,15 +1,19 @@
 const express = require('express');
 const { getTopics } = require('./controllers/topicController');
 const { getApiEndPoints } = require('./controllers/apiController');
+const { getArticleById } = require('./controllers/articleController');
 
 const app = express();
 
+app.get('/api/articles/:article_id', getArticleById);
 app.get('/api/topics', getTopics);
 app.get('/api', getApiEndPoints);
 
 app.use((err, req, res, next) => {
   if (err.code === '22P02') {
     res.status(400).send({ msg: 'Bad Request' });
+  } else if (err.status) {
```

Comment:
If you specify on line 16 you want to send the err.msg, it's generally considered good form to include this in the else if - e.g. `(err.status && err.msg)`